R/pkg/R/mllib.R

```diff
@@ -108,28 +107,32 @@ setMethod(
 #' @param maxit Integer giving the maximal number of IRLS iterations.
 #' @return a fitted generalized linear model
 #' @rdname glm
+#' @name glm
 #' @export
 #' @examples
 #' \dontrun{
 #' sc <- sparkR.init()
 #' sqlContext <- sparkRSQL.init(sc)
 #' data(iris)
 #' df <- createDataFrame(sqlContext, iris)
-#' model <- glm(Sepal_Length ~ Sepal_Width, df, family="gaussian")
+#' model <- glm(Sepal_Length ~ Sepal_Width, df, family = "gaussian")
 #' summary(model)
 #' }
 setMethod("glm", signature(formula = "formula", family = "ANY", data = "SparkDataFrame"),
           function(formula, family = gaussian, data, epsilon = 1e-06, maxit = 25) {
             spark.glm(data, formula, family, epsilon, maxit)
           })
```

Comment:
Because ```glm``` is R-compliant function, so I left the argument names consistent with native R.